platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/transition/WorkflowTransitionActionPermission.java

```diff
@@ -56,18 +56,18 @@ public boolean test(R tableRow, IBasicUser user) {
 	 * @param user
 	 *            the user that want to execute the transition (never
 	 *            <i>null</i>)
-	 * @return true if the current user is in one of the roles or if a user that
-	 *         delegated to him is in one of the roles (or has set him as
-	 *         substitute)
+	 * @return true if the current user is in one of the permissions or if a
```

Comment:
That `@return` description appears nonsensical now. Also, we should avoid "him".

Reply:
changed to 
```
	 * @return <i>true</i> if the given user has this permissions or if a user
	 *         that delegated the task to the user (or defined the user as
	 *         substitute) has this permission
```